ui/v2.5/src/hooks/Lightbox/Lightbox.tsx

```diff
@@ -260,6 +304,38 @@ export const LightboxComponent: React.FC<IProps> = ({
                 {`${currentIndex + 1} / ${images.length}`}
               </b>
             </div>
+            {slideshowEnabled && (
+              <>
+                <div className={CLASSNAME_DELAY}>
+                  <InputGroup>
+                    <FormLabel column sm="5">
+                      Delay (Sec)
+                    </FormLabel>
+                    <Col sm="4">
+                      <FormControl
+                        type="number"
+                        value={(slideshowInterval ?? 0) / SECONDS_TO_MS}
+                        onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+                          handleSlideshowIntervalChange(
+                            Number.parseInt(e.currentTarget.value, 10) *
+                              SECONDS_TO_MS
+                          );
+                        }}
+                        size="sm"
+                        disabled={slideshowInterval === null}
+                      />
```

Comment:
Please add `className="text-input"` so that the text field styling is consistent with fields.

Reply:
Added className + set a min value of 1.

- re: left/right click - I have a fix in mind. Will implement later today.
- re: 0/negative number - Does go a bit bonkers :) the min setting on the input should address that
- re: drag input - that works only on the spinner itself. If you click on the spinner and drag up/down, the number will change as expected. Dragging the value, I believe, is not built into number inputs and is a whole other implementation. Not sure it's worth getting into. WDYT?
- re: move between tabs - this is because the interval is allowed to run in the background, while the UI update is not. This can be solved using the [Page Visibility API](https://developer.mozilla.org/en-US/docs/Web/API/Page_Visibility_API). Pause on lost visibility and restart on the visible callback

I'll take care of the other issues later today.
Thanks for the feedback!

Reply:
Done and done.

Reply:
I can still type in `0` or a negative number and it goes bonkers, so `min` won't be enough.

My suggestion would be the following:
* have the slideshow be the default behaviour when clicking an image in the wall view and remove the `Start Slideshow` menu item. Clicking the image shows the slideshow but starts paused.
* delay should always be shown, even if paused. It currently sets the delay text field to 0. It should keep the existing value. It's not necessary to disable the text field.
* `useInterval` should ensure that the interval value is not less than some safe value. It should either error out or choose a safe value if an invalid value is provided. The slideshow client should also set the minimum value as `1` so that even if the user types in a `0` or negative value, it will still use a delay of `1`.
* the styling needs to be fixed up on smaller screens, compare non-slideshow view vs slideshow view on viewports < 800px wide:

![image](https://user-images.githubusercontent.com/53250216/113243590-0ac60f00-92ff-11eb-83ba-57361ee54731.png)

In terms of the drag input, I found that if I highlight the current value and don't drag outside the textfield, then it will highlight the input value correctly, if I let go outside the textfield then the slideshow ends. To be clear, all I'm trying to do is select the current value to type in a new value (which can also be done by double clicking on the field). I'm not sure if there's a fix or not.

Reply:
I was initially trying to make the slideshow feature as isolated as possible, but I'm on board with what you're suggesting.  
I'll also add better handling of invalid values both on the input and useInterval and look into < 800px viewport width.

Regarding the drag behavior, if I understand the use-case you're describing correctly, this is due to how the Lightbox handles mouse events and not specifically the input. I'll see how much of an effort it'll be to fix this and go from there.

Reply:
- I implemented your suggestion for how the slideshow should work

- I added checks for invalid values both on the input itself and the useInterval hook

- Space comes at a premium for < 800px, so I created a CSS breakpoint and replaced the inline form with an icon and a popover form. I also refactored the HTML, so the entire header isn't so much based on margins anymore

- Regarding the input, I fixed it so that if the mousedown event happens within the input, it's not going to close the lightbox. It's not perfect, but I think it's much better (and less annoying) than it was

Reply:
> * Regarding the input, I fixed it so that if the mousedown event happens within the input, it's not going to close the lightbox. It's not perfect, but I think it's much better (and less annoying) than it was

I can no longer click on the field to edit it. Clicking on the field shows the cursor briefly, then it defocuses.

Everything else looks good.

Reply:
Sorry, had a super busy few days.
I'll take a look at the input fix.

Reply:
All good. I've fixed the issue.

Reply:
Great! btw: got BE and FE going fully native on Apple M1, FYI :)